docs/readme.md

```diff
@@ -0,0 +1,75 @@
+# jackpocket-ops-protobuf
```

Comment:
Looks like you copied the readme from the protobuf library? Need to get the right one here.